lib/snail.rb

```diff
@@ -89,12 +89,16 @@ def origin
     @origin ||= Snail.home_country
   end
 
-  def to_s
-    [name, line_1, line_2, city_line, country_line].select{|line| !(line.nil? or line.empty?)}.join("\n")
+  def to_s(country: false)
```

Comment:
let's go with `nil` for the default, since it's not a strict false. maybe someone will come up with a use case for always suppressing the country?